dotnetv3/dynamodb/README.md

```diff
@@ -1,465 +1,143 @@
-<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-     SPDX - License - Identifier: Apache - 2.0 -->
-# Amazon DynamoDB code examples in C\#
+# DynamoDB code examples for .NET
+
+## Overview
+
+The code examples in this directory demonstrate how to work with Amazon DynamoDB using the AWS SDK for JavaScript version 3 (v3).
+
+Amazon DynamoDB is a key-value and document database that delivers single-digit millisecond performance at any scale. It's a fully managed, multiregion, multimaster, durable database with built-in security, backup and restore, and in-memory caching for internet-scale applications.
+
+## ⚠️ Important
+
+- Running this code might result in charges to your AWS account. 
+- We recommend that you grant your code least privilege, or at most the minimum
+  permissions required to perform the task. For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+- This code is not tested in all AWS Regions. For more information, see 
+  [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/).
+
+## Code examples
+
+### Scenario
+
+  [DynamoDB Basics](scenarios/DynamoDB_Basics/DynamoDB_Basics_Scenario/)
```

Comment:
Update the title to be consistent with other languages: Get started using DynamoDB tables, items, and queries
(see the [DynamoDB user guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/example_dynamodb_Scenario_GettingStartedMovies_section.html)).

Reply:
changed